### PR TITLE
feat(post): add afup day 2022

### PR DESCRIPTION
**Pull request by fferriere**

### CODE REVIEW

File: _posts/2022-05-30-afup-day-lille-2022.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: Retour sur l'AFUP Day Lille 2022
+description: Une partie de notre équipe backend s'est rendu à l'AFUP Day Lille 2022
```
Comment (FloFrad) [inline _posts/2022-05-30-afup-day-lille-2022.md:4]:
```suggestion
description: Une partie de notre équipe backend s'est rendue à l'AFUP Day Lille 2022
```

File: _posts/2022-05-30-afup-day-lille-2022.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+Les [*constantes*](https://www.php.net/manual/en/language.oop5.constants.php) peuvent maintenant être finales, cela bloquera la possibilité de surcharger leurs valeurs par héritage.
+
+Il est désormais possible de faire des propriétés [*readonly*](https://www.php.net/manual/en/language.oop5.properties.php#language.oop5.properties.readonly-properties). Cette propriété devra forcément être typée, et ne pourra pas avoir de valeur par défaut. *readonly* ne peut pas être utilisé avec *static*. Si la propriété est un objet, l'instance pourra être modifiée (par exemple avec des setters), mais pas remplacée.
+
+Il est maintenant possible d'instancier des valeurs par défaut. Par exemple :
+
+```php
+function serialize(string \$data, Formatter \$formatter = new
+DefaultFormatter()) { ... }
+```
```
Comment (FloFrad) [inline _posts/2022-05-30-afup-day-lille-2022.md:43]: **suggestion:** multi-line to increase readability
```suggestion
```php
function serialize(
    string \$data, 
    Formatter \$formatter = new DefaultFormatter()
) { ... }
```
```

File: _posts/2022-05-30-afup-day-lille-2022.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+## Les subtilités du e-commerce à la française
+
+Après nous avoir présenté les différentes taxes françaises et chez quelques-uns de nos voisins européens, [*David Buros*](https://twitter.com/lanfisis) nous a résenté les différents problèmes qu'il a rencontrés avec Sylius:
+
+- l'affichage du prix HT et du prix TTC
+- la gestion des réductions avec ce double affichage
+- la gestion des écotaxes
+- le paiement par mandat administratif
+
+## Comment on est passé de 1800 emails à 70 000 par jour chez Trustt en 1 mois avec RabbitMQ
```
Comment (FloFrad) [inline _posts/2022-05-30-afup-day-lille-2022.md:87]: Choose your style 😄 
```suggestion
## Comment on est passé de 1 800 emails à 70 000 par jour chez Trustt en 1 mois avec RabbitMQ
```
```suggestion
## Comment on est passé de 1800 emails à 70000 par jour chez Trustt en 1 mois avec RabbitMQ
```
```suggestion
## Comment on est passé de 1.800 emails à 70.000 par jour chez Trustt en 1 mois avec RabbitMQ
```

File: _posts/2022-05-30-afup-day-lille-2022.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+## Pourquoi vous n'attirerez et ne retiendrez pas les femmes dans vos équipes tech.
+
+[*Marcy Charollois*](https://fr.linkedin.com/in/marcy-ericka-charollois-1604) fait un constat sur le monde du travail dans le numérique, dominé largement par les hommes et ne laissant que trop peu de place aux femmes. Marcy commence par introduire la notion d'habitus, qui désigne un système de préférences de style de vie particulier à chacun qui influe les pratiques des individus au quotidien. Ces pratiques sont intériorisées inconsciemment, car l'individu s'adapte et s\'intègre à son environnement social. Cela crée un groupe majoritaire qui devient décisionnaire. Se met alors en place un statu quo qui va soit inclure ou exclure et qui est fortement dominé par la pensée masculine. Marcy nous dévoile que sur 100 % de freins ressentis par les femmes dans la tech, 30% proviennent des biais d\'oppression de groupe, une part donc assez conséquente.
```
Comment (FloFrad) [inline _posts/2022-05-30-afup-day-lille-2022.md:107]:
```suggestion
[*Marcy Charollois*](https://fr.linkedin.com/in/marcy-ericka-charollois-1604) fait un constat sur le monde du travail dans le numérique, dominé largement par les hommes et ne laissant que trop peu de place aux femmes. Marcy commence par introduire la notion d'habitus, qui désigne un système de préférences, de style de vie particulier à chacun, qui influence les pratiques des individus au quotidien. Ces pratiques sont intériorisées inconsciemment, car l'individu s'adapte et s\'intègre à son environnement social. Cela crée un groupe majoritaire qui devient décisionnaire. Se met alors en place un statu quo qui va soit inclure ou exclure et qui est fortement dominé par la pensée masculine. Marcy nous dévoile que sur 100 % de freins ressentis par les femmes dans la tech, 30% proviennent des biais d\'oppression de groupe, une part donc assez conséquente.
```

File: _posts/2022-05-30-afup-day-lille-2022.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+Le constat est réel, les femmes dans la tech vivent mal leur condition de femmes, il faut changer ce sentiment, mais les attitudes face au changement sont variées. 15% de personnes sont réfractaires, il sera donc difficile de faire évoluer les choses avec eux, 15% sont déjà partantes et 70% sont neutres, potentiellement pour ce changement, mais ne savent pas comment le faire.
+
+Marcy nous donne alors des clés qui permettront d'attirer les femmes dans nos équipes en mettant en avant les freins ressentis par les femmes : une expression du genre, une légitimité face au métier exercée, des a priori sur la provenance des profils féminins qui sont souvent reconvertis et donc potentiellement juniors :
```
Comment (FloFrad) [inline _posts/2022-05-30-afup-day-lille-2022.md:111]:
```suggestion
Marcy nous donne alors des clés qui permettront d'attirer les femmes dans nos équipes en mettant en avant les freins ressentis par celles-ci : une expression du genre, une légitimité face au métier exercé, des a priori sur la provenance des profils féminins qui sont souvent reconvertis et donc potentiellement juniors :
```

File: _posts/2022-05-30-afup-day-lille-2022.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+La conférence continue sur les actions à mener pour garder les femmes dans nos équipes :
+
+- parler d'égal à égal pour éviter la posture sachant(e)/ignorante
+- soyez clair, transparent sur les salaires, les évolutions de poste
+- mettez en place des moments conviviaux plus portés sur des préférences féminines
+- minimiser les interruptions pendant les prises de parole
+- écoutez des besoins spécifiques inhérents aux femmes et accommodez-les en offrant des ressources sans juger : parentalité, menstruation, assistance psychologique, adaptation à l'emploi du temps
+- encouragez les femmes à prendre la parole, à devenir un rôle modèle parce que compétente
+
+Marcy termine sa conféren ce par la présentation de quelques chiffres sur l'évolution de carrière des femmes et des enjeux psychosociaux résultant
```
Comment (FloFrad) [inline _posts/2022-05-30-afup-day-lille-2022.md:126]:
```suggestion
Marcy termine sa conférence par la présentation de quelques chiffres sur l'évolution de carrière des femmes et des enjeux psychosociaux résultant
```

File: _posts/2022-05-30-afup-day-lille-2022.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+- ***User journey*** : identification des différentes étapes (déterminées suite au bench et entretiens), mise en place du workflow
+- ***Specs fonctionnelles et design*** : cas d'usages, règles métier, versions, KPI / création de wireframes, maquettes UI et prototypes
+
+Cette méthodologie comprend cependant des limites ! Elle prend beaucoup de temps et est soumise au contexte, aux priorisations d'autres features, au downsizer...
+
+## Le Prométhée moderne : embarquer PHP dans Go
+
+[*Kevin Dunglas*](https://twitter.com/dunglas) nous a parlé de comment embarquer PHP dans du Go. Après avoir listé les différentes SAPI (Module Apache, FPM, ...) et nous avoir expliqué brièvement le langage Go et sa librairie standard net/http, Kevin nous a présenté **FrankenPHP** et toute la réflexion et les contraintes rencontrées pour le créer. Ce projet est un nouveau serveur Web en Go qui est capable d'appeler l'interpréteur PHP et donc de faire tourner nos applications Web. Le projet est bien avancé, mais pas terminé. N'hésitez pas à le contacter si vous voulez essayer avec vos applications, les retours lui seront utiles. En tout cas, chez Bedrock, on va suivre ça de près !
```
Comment (FloFrad) [inline _posts/2022-05-30-afup-day-lille-2022.md:76]:
```suggestion
[*Kévin Dunglas*](https://twitter.com/dunglas) nous a parlé de comment embarquer PHP dans du Go. Après avoir listé les différentes SAPI (Module Apache, FPM, ...) et nous avoir expliqué brièvement le langage Go et sa librairie standard net/http, Kévin nous a présenté **FrankenPHP** et toute la réflexion et les contraintes rencontrées pour le créer. Ce projet est un nouveau serveur Web en Go qui est capable d'appeler l'interpréteur PHP et donc de faire tourner nos applications Web. Le projet est bien avancé, mais pas terminé. N'hésitez pas à le contacter si vous voulez essayer avec vos applications, les retours lui seront utiles. En tout cas, chez Bedrock, on va suivre ça de près !
```

**Review by FloFrad — APPROVED** [review]
👏

**Review by emyrtille — APPROVED** [review]

File: _posts/2022-05-30-afup-day-lille-2022.md (hunk 3 above)
Comment (fferriere) [reply _posts/2022-05-30-afup-day-lille-2022.md:87]: You're right, we copy/paste the conference's name :shrug:

File: _posts/2022-05-30-afup-day-lille-2022.md (hunk 7 above)
Comment (fferriere) [reply _posts/2022-05-30-afup-day-lille-2022.md]: done

File: _posts/2022-05-30-afup-day-lille-2022.md (hunk 4 above)
Comment (fferriere) [reply _posts/2022-05-30-afup-day-lille-2022.md]: done

File: _posts/2022-05-30-afup-day-lille-2022.md (hunk 3 above)
Comment (fferriere) [reply _posts/2022-05-30-afup-day-lille-2022.md]: done

File: _posts/2022-05-30-afup-day-lille-2022.md (hunk 2 above)
Comment (fferriere) [reply _posts/2022-05-30-afup-day-lille-2022.md]: done

**Review by omansour — APPROVED** [review]

File: _posts/2022-05-30-afup-day-lille-2022.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+Une des nouveautés phare de la version 8.1 est l'ajout de la gestion des [*enums*](https://www.php.net/manual/en/language.types.enumerations.php). Une énumération est un [*objet*](https://www.php.net/manual/en/class.unitenum.php) et non un type scalaire. Il est donc possible de l'étendre ([*via des interfaces*](https://www.php.net/manual/en/language.enumerations.methods.php), ...). Cela veut aussi dire que nous ne pouvons pas les utiliser comme clefs de tableau par exemple. Il est par contre possible d'utiliser la propriété \$myEnum-\>value. Attention, les énumérations et les classes partagent le même espace de nom, nous ne pouvons donc pas avoir une enum et une classe s'appelant pareil dans le même namespace.
+
+Les Fibers, solution pour rendre les programmes interruptibles, a été rapidement introduite, mais nous vous conseillons de regarder la [*conférence sur ce sujet*](https://afup.org/talks/3794-fiber-la-porte-ouverte-sur-l-asynchrone) de notre cher et estimé collègue [*Benoit Viguier*](https://twitter.com/b_viguier).
```
Comment (b-viguier) [inline _posts/2022-05-30-afup-day-lille-2022.md:30]: :kissing_heart:

**Pull request merged**